site/content/blog/carvelizing-helm-chart.md

```diff
@@ -0,0 +1,293 @@
+---
+title: "Carvelizing Helm Chart"
+slug: carvelize-helm-chart
+date: 2022-02-16
+author: Rohit Aggarwal
+excerpt: "Use Carvel to author and consume helm chart "
+image: /img/logo.svg
+tags: ['carvel', 'helm', 'gitops']
+---
+
+In this blog post we will first show you how to wrap and distribute [`Bitnami nginx helm chart`](https://github.com/bitnami/charts/tree/master/bitnami/nginx) as a Carvel package, and then install it on the Kubernetes cluster via PackageInstall CR (via kapp-controller).
+
+## Why should I choose Carvel
+
+Kubernetes configuration takes many forms – plain YAML configurations, Helm charts, ytt templates, jsonnet templates, etc. Software running on Kubernetes lives in many different places: a Git repository, an archive over HTTP, a Helm repository, etc.
+
+Kapp-controller, a Carvel tool, provides software authors flexibility to choose their own configuration tools, while providing software consumers with consistent declarative APIs to customize, install, and update software on Kubernetes from various sources.
+
+## Prerequisites
+
+Basic knowledge of imgpkg, kbld, kapp-controller, kapp
+
+[`imgpkg`](https://carvel.dev/imgpkg/): A tool to package, distribute, and relocate your Kubernetes configuration and dependent OCI images as one OCI artifact: a bundle.
+
+[`kbld`](https://carvel.dev/kbld/): kbld incorporates image building and image pushing into your development and deployment workflows.
+
+[`kapp-controller`](https://carvel.dev/kapp-controller/): kapp-controller provides declarative APIs to create, customize, install, and update your Kubernetes applications into packages.
```

Comment:
imo worth adding a second sentence here trying to capture some of the gitops marketplace hearts & minds,  maybe something like "It also continuously fetches and reconciles resources to converge into desired state you have declared."  ?

Reply:
Adopted